packages/api/db/reset.sql

```diff
@@ -15,4 +15,6 @@ DROP TABLE IF EXISTS public.user CASCADE;
 DROP TABLE IF EXISTS cargo.aggregate_entries;
 DROP TABLE IF EXISTS cargo.aggregates;
 DROP TABLE IF EXISTS cargo.deals;
-DROP SERVER IF EXISTS dag_cargo_server CASCADE;
\ No newline at end of file
+DROP SERVER IF EXISTS dag_cargo_server CASCADE;
+
+ALTER DATABASE postgres RESET ALL;
```

Comment:
Was this meant to be included? If so, can we please get a comment to explain why?